.github/workflows/stale.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale pull requests"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.'
+        exempt-pr-labels: 'awaiting-approval,work-in-progress'
+        stale-pr-label: stale
```

Comment:
@kgyrtkirk , should we use the `days-before-stale` and `days-before-close` configs? Maybe closing them automatically after 7 days?

Reply:
I thinked that we could use the defaults which are 60/7 respectively
https://github.com/actions/stale/blob/master/action.yml
should I add them to make it clearer?

Reply:
Oh, I did not realize that. Yes, I think making them explicit may be better.